### PR TITLE
Newsletter block

### PR DESCRIPTION
**Pull request by duarte**
Adds the newsletter sign-up block!

Still needs small tweaks, and the email input is not styled — in fact the actual form does not point to the right place yet…

CC @gidsg @mr-mr

### CODE REVIEW

File: src/assets/imgs/theguardian-documentaries.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (mr-mr) [inline src/assets/imgs/theguardian-documentaries.svg]: How is this .svg being placed in the page?
We should add this through the _mixin file in scss if it's not.

**Comment by mr-mr** [comment]
Aside from that 👍

File: src/css/_onward.scss [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+    max-width: none;
+  }
+  @include mq(desktop) {
+    width: 940px;
+  }
+  @include mq(leftCol) {
+    width: 1100px;
+  }
+  @include mq(wide) {
+    // @include gs-span(13);
```
Comment (gidsg) [inline src/css/_onward.scss]: Can you delete this or comment in?

**Comment by gidsg** [comment]
👍 (aside from comments)

**Pull request merged**